CMakeLists.txt

```diff
@@ -719,7 +719,7 @@ if(AVIF_BUILD_APPS OR (AVIF_BUILD_TESTS AND (AVIF_ENABLE_FUZZTEST OR AVIF_ENABLE
     # avif_apps_internal is to use when linking to avif_internal.
     if(BUILD_SHARED_LIBS)
         add_library(avif_apps_internal STATIC)
-        target_link_libraries(avif_apps_internal PUBLIC avif_internal PRIVATE avif_apps_obj)
+        target_link_libraries(avif_apps_internal PRIVATE avif_apps_obj PUBLIC avif_internal)
```

Comment:
The order in which `avif_apps_ob` and `avif_internal` are listed may not matter. I can revert this change if you prefer.

Reply:
Having them in alphabetical order is fine. I wonder how consistent other entries are.